servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/DefaultNettyConnection.java

```diff
@@ -921,6 +929,12 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
             LOGGER.debug("{} Received a user event: {}", ctx.channel(), evt);
             if (evt == CloseHandler.InboundDataEndEvent.INSTANCE) {
                 connection.nettyChannelPublisher.channelOnComplete();
+                if (connection.readOnChannelComplete) {
+                    // h2 child channel may require read() to notify of channel closure. This constraint is lifted
+                    // with netty enhancement [1] but keep this for some time to give folks time to update Netty.
+                    // [1] https://github.com/netty/netty/pull/13779
```

Comment:
related to https://github.com/netty/netty/pull/13779

Reply:
Consider prefixing this comments with `// FIXME 0.43` to make sure we won't forget to remove it in the future